src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -218,6 +221,11 @@ internal static Summary[] Run(BenchmarkRunInfo[] benchmarkRunInfos)
                 }
             }
 
+            if (RuntimeInformation.IsWindows())
+            {
+                Console.Title = string.Empty;
```

Comment:
Instead of setting the title to empty, you should cache the old title before the run, then reset it back to the old title after the run.

```cs
var oldTitle = Console.Title;
...
Console.Title = oldTitle;
```

Reply:
I actualy thought about this, I don't think we set the a title before runing the benchmark, or after, that is why I did not cache it.

But this is in fact more future proof solution, I will change it

Reply:
It's because other applications can run in the same console window.